popSimManu.tex

```diff
@@ -528,57 +528,92 @@ \subsection*{Model QC procedures}
+population size instantaneously decreases from $N=10^4$ to $N=10^3$ 500 
+generations before the present. Each HomSap simulation was run
+using the HapmapII\_GRCh37 genetic maps. For \emph{D. melanogaster}
+we estimated $N(t)$ from an African sample simulated under the DroMel, 
+African3Epoch\_1S16 model using the Comeron2012\_dm6 map. Finally we ran
+simulations of \emph{A. thaliana} genomes using the AraTha African2Epoch\_1H18
+model under the Salome2012\_TAIR7. For each model, 3 replicate whole genomes were
+simulated and the population size estimated from those data. In all cases we
+set the sample size of the focal population to $N=50$ chromosomes. 
+
+Following simulation, low recombination portions of chromosomes were masked
+from the analysis in a manner that reflects the ``accesible" subset of sites
+used in empirical population genomic studies \citep[e.g.,][]{danecek20111000,langley2012genomic}.
+Specifically we masked regions of 1cM or greater in the lowest 5th percentile of the empirical
+distribution of recombination.  
 
-
-
-\subsection*{Workflow for two population analysis}
 Our goal for the two population analysis pipeline was to explore two population inference
 on some of the multi-population demographic models implemented in \stdpopsim.
-The models we examined include the human Gutenkunst model \citep{gutenkunst2009inferring} as
-well as the \emph{Drosophila} Li and Stephan model \citep{li2006inferring}. The human
-model is a three population model (Africa, Europe, and Asia) including post-divergence
+For HomSap we examined the OutOfAfrica_3G09 model with the HapmapII\_GRCh37 genetic map,
+and for DroMel we simulated under the OutOfAfrica_2L06 model with the Comeron2012\_dm6.
+The HomSap model is a three population model (Africa, Europe, and Asia) including post-divergence
 migration and exponential growth (Figure \ref{fig:IM_popn_human}C), whereas the
-\emph{Drosophila} model is a two population model (Africa and Europe) with no post-divergence
+DroMel model is a two population model (Africa and Europe) with no post-divergence
 migration and constant population sizes (Figure \ref{fig:two_popn_fly}).
 
-To conduct inference on these models, we implemented three commonly-used approaches:
+To conduct inference on these models, we used three commonly used methods:
 \dadi~\citep{gutenkunst2009inferring}, \fastsimcoal2~\citep{excoffier2013robust},
-and \smcpp~\citep{terhorst2017robust}. Our aim was not to exhaustively
-test the performance of these methods, but rather to explore their behavior
-in two different inference scenarios and demonstrate the potential for this resource to
-eventually be used for more in-depth benchmarking. As a result, these methods were used
+and \smcpp~\citep{terhorst2017robust}. As above, these methods were used
 generally with default settings and we did not attempt to optimize their performance or fit
 parameter-rich demographic models. As with the single population inference pipeline,
```

Comment:
could probably lump the snakemake sentence into the previous one since theres some redundancy with "as above..."